src/main/java/frc/robot/RobotContainer.java

```diff
@@ -38,16 +40,9 @@ private void configureBindings() {
             * DrivetrainSubsystem.MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND));
 
     backButton.onTrue(new InstantCommand(drivetrainSubsystem::zeroGyroscope));
-
-    bButton.toggleOnTrue(new DefaultDriveCommand(
-        drivetrainSubsystem,
-        () -> -mod.modifyAxis(driverController.getRawAxis(Constants.LEFT_Y_AXIS))
-            * DrivetrainSubsystem.MAX_VELOCITY_METERS_PER_SECOND,
-        () -> -mod.modifyAxis(driverController.getRawAxis(Constants.LEFT_X_AXIS))
-            * DrivetrainSubsystem.MAX_VELOCITY_METERS_PER_SECOND,
-        () -> -mod.modifyAxis(driverController.getRawAxis(Constants.RIGHT_X_AXIS))
-            * DrivetrainSubsystem.MAX_ANGULAR_VELOCITY_RADIANS_PER_SECOND,
-        new Rotation2d(Constants.DRIVE_ROTATION_TARGET_DEGREES)));
+    
+    yButton.onTrue(new ToggleRotationTarget(drivetrainSubsystem, vision.getXOffset()));
+    bButton.onTrue(new ToggleRotationTarget(drivetrainSubsystem, Rotation2d.fromDegrees(Constants.DRIVE_ROTATION_TARGET_DEGREES)));
```

Comment:
We could convert each of these commands to a toggle like you suggested. the first press sets the target and the second press disables the rotation target